### PR TITLE
Add Readme for frontend roadmap

### PR DESCRIPTION
**Pull request by vasucp1207**
Take a look at the readme and please provide me feedback on what can I improve more.

### CODE REVIEW

**Comment by siddhant-khisty** [comment]
@vasucp1207 Why are there 305 files? only one file had to be added

**Comment by vasucp1207** [comment]
@siddhant-khisty, they all are in .history folder

**Comment by siddhant-khisty** [comment]
@vasucp1207 I'm not sure how a history folder was created, but please delete it. This PR should only add a `roadmap.md` to the Frontend-Roadmap folder

**Comment by vasucp1207** [comment]
Hey, @siddhant-khisty sorry for that I delete the .history folder

File: Frontend-Development/roadmap.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+#Frontend Roadmap
```
Comment (siddhant-khisty) [inline Frontend-Development/roadmap.md:1]: Add spaces between every heading for better readability

File: Frontend-Development/roadmap.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+- [Tutorial](https://www.youtube.com/watch?v=-qfEOE4vtxE)
+- [Docs](https://getbootstrap.com/docs/5.2/getting-started/introduction/)
+
+### Tailwind CSS
+- [Tutorial 1](https://www.youtube.com/watch?v=UBOj6rqRUME)
+- [Tutorial 2](https://www.youtube.com/watch?v=dFgzHOX84xQ)
+- [Docs](https://tailwindcss.com/docs/installation)
+
+## What is PWA(Progressive Web Apps)
+### PWA
```
Comment (siddhant-khisty) [inline Frontend-Development/roadmap.md:113]: Remove line 113 as it is not required. Only keep the resources

**Review by siddhant-khisty — CHANGES_REQUESTED** [review]

**Comment by vasucp1207** [comment]
I think we should give it the name README.md not something like roadmap.md because this is not shown by default in the roadmap folder.

**Comment by kaiwalyakoparkar** [comment]
> I think we should give it the name README.md not something like roadmap.md because this is not shown by default in the roadmap folder.

Hmm.. nice idea, let's do that

**Comment by vasucp1207** [comment]
@kaiwalyakoparkar, yeah I do that.

**Review by siddhant-khisty — APPROVED** [review]
LGTM

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm

**Pull request merged**